sdk/spring-sdk/src/main/java/kalix/springsdk/annotations/Publish.java

```diff
@@ -32,4 +32,19 @@
     /** Assign the name of the topic to be used for eventing out. */
     String value();
   }
+
+
+  /**
+   * Annotation for marking a method as producing information to be published on a PubSub or Kafka
```

Comment:
Direct stream, not PubSub or Kafka.

Reply:
```suggestion
   * Annotation to configure the component to publish an event stream to other Kalix services.
```

Reply:
Just realised that it should not be possible to use this annotation in anything other than an Action and it should be combined with a subscription to an Event Sourced entity. 

Btw, the text here comes directly from the proto annotation in the proxy. I copied it blindly. 

That's also not correct. This is an annotation that is only valid at service level. We should not mention `method`.